@xen-orchestra/fs/src/abstract.js

```diff
@@ -410,6 +410,21 @@ export default class RemoteHandlerAbstract {
     await this._unlink(file).catch(ignoreEnoent)
   }
 
+  async write(
+    file: File,
+    buffer: Buffer,
+    position: number
+  ): Promise<{| bytesWritten: number, buffer: Buffer |}> {
+    await timeout.call(
```

Comment:
We should remove this timeout, the timeout only applied to instantaneous operations (`O(1)`) which is not the case here because it depends of the size of the data.